tool-labs/accounteligibility/events.php

```diff
@@ -22,6 +22,46 @@ public function getDefaultEventID()
      */
     public function getEvents()
     {
+
+        ##########
+        ## 2017: Wikimedia Foundation elections
+        ##########
+        yield (new Event(44, 2017, 'Wikimedia Foundation elections', '//meta.wikimedia.org/wiki/Wikimedia Foundation elections/2017'))
+            ->addRule(new NotBlockedRule(1), Workflow::HARD_FAIL)// not blocked on more than one wiki
+            ->addRule(new NotBotRule(), Workflow::HARD_FAIL)
+            ->addRule(new EditCountRule(300, null, '<20170401', EditCountRule::ACCUMULATE))// 300 edits before 1 April 2017
+            ->addRule(new EditCountRule(20, '20161001', '<20170401', EditCountRule::ACCUMULATE))// 20 edits between 1 October 2016 and 1 April 2017
```

Comment:
Sorry, one account as in one 'global' account but those 20 edits could be from multiple wikis and be added together.